common/remote/hooks/libarchive/archive.cpp

```diff
@@ -608,7 +608,7 @@ class CArchiveFileHook : public CInterface, implements IContainedFileHook
 
 extern ARCHIVEFILE_API void installFileHook()
 {
-    SpinBlock b(*lock); // Probably overkill!
+    CriticalBlock b(*lock); // Probably overkill!
```

Comment:
Prior thinking was that spinlocks were faster than critsecs where contention was very unlikely. Did you mean critsec or spinlock in your first paragraph above?